source/includes/live-upgrade.rst

```diff
@@ -0,0 +1,4 @@
+Starting in ``mongosync`` 1.7.0, you can upgrade ``mongosync`` without
+restarting data synchronization operations from the beginning. You can
+only live upgrade from ``mongosync`` 1.5.0 or later to ``mongosync``
```

Comment:
```suggestion
only live upgrade from ``mongosync`` 1.6.0 or later to ``mongosync``
```
This also needs to be updated in the staged 1.7.0 [release notes](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-29505-live-update/release-notes/1.7/#std-label-1.7.0-c2c-release-notes)